packages/contracts/contracts/optimistic-ethereum/OVM/bridge/tokens/OVM_L1ETHGateway.sol

```diff
@@ -68,66 +67,103 @@ contract OVM_L1ETHGateway is iOVM_L1ETHGateway, OVM_CrossDomainEnabled, Lib_Addr
      * Depositing *
      **************/
 
+    /**
+     * @dev This function can be called with no data
+     * to deposit an amount of ETH to the caller's balance on L2.
+     */
     receive()
         external
         payable
     {
-        _initiateDeposit(msg.sender, msg.sender);
+        _initiateDeposit(msg.sender, msg.sender, 0, bytes(""));
     }
```

Comment:
Without a default gas value, `receive()` will no longer work without some data passed to it, at which point you might as well just specify the `deposit()` function.